reascripts/common/scripts/png_to_lua.rb

```diff
@@ -19,11 +19,11 @@
   f.write("  width = #{image.width},\n")
   f.write("  height = #{image.height},\n")
 
-  f.write("  bytes = table.concat({\n")
-  image.to_blob.bytes.each_slice(100) do |slice|
-    f.write("string.char(#{slice.join(",")}),\n")
-  end
-  f.write("  })\n")
+  bytes = image.to_blob.bytes.map { |b|
+    format("\\%d", b)
+  }.join
```

Comment:
just command-tabbed my way to vscode and this was line was on my screen and i felt a sincere sense of loss that the return value of `format("\\%d")` isn't a function expecting a byte argument

Reply:
Yeah, Ruby's FP support is lackluster at times. There is a convenient shortcut for no-argument methods, which could be used/abused along with Ruby's open classes:

```
class Integer
  def code_string
    return format("\\%d", self)
  end
end

bytes = blob.map(&:code_string).join
```

Reply:
> could be used/**abused** along with Ruby's open classes

me at ruby jobs in the past feeling pretty seen right now